src/web/app/pages-admin/admin-notifications-page/notification-edit-form/notification-edit-form.component.scss

```diff
@@ -19,7 +19,7 @@
   margin-top: 20px;
 }
 
-.session-form-buttons > button {
+.notification-form-buttons > button {
```

Comment:
I believe this can be deleted since you deleted the the row `notification-edit-form.component.html`(line 8)